src/main.rs

```diff
@@ -32,3 +40,11 @@ async fn main() -> Result<()> {
 
     Ok(())
 }
+
+#[cfg(not(target_arch = "wasm32"))]
+#[cfg(not(feature = "cli"))]
+#[tokio::main]
+pub async fn main() -> Result<()> {
```

Comment:
don't do this, this feature isn't useful

Reply:
same